renderer/routes/recent.svelte

```diff
@@ -0,0 +1,110 @@
+<script>
+  import { onMount } from 'svelte'
+  import { fade } from 'svelte/transition'
+  import { _ } from 'svelte-intl'
+  import { replace } from 'svelte-spa-router'
+  import { distinct, filter, map } from 'rxjs/operators'
+  import { Heading, Button, PlaylistTracksTable } from '../components'
+  import { load, changes, removals } from '../stores/playlists'
+  import { add } from '../stores/track-queue'
+  import {
+    invoke,
+    formatTimeLong,
+    fromServerChannel,
+    sumDurations
+  } from '../utils'
+
+  let playlist
+
+  $: playlistId = 151120
+  $: if (!playlist || playlist.id !== playlistId) {
+    loadPlaylist()
+  }
```

Comment:
For our list of recent tracks, since there is no id in url, we can invoke `tracks.getRecent()` service method inside `onMount()`.

For playlist, it's a bit different, because we load them from their id, which comes from the url, and we need to check on every render whether the url value has changed or not.